engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/views/register.html

```diff
@@ -15,17 +15,22 @@ <h2 translate>Content Host Registration</h2>
       </li>
       <li>
         <p translate>Install the pre-built bootstrap RPM:</p>
-        <pre><code>rpm -Uvh http://{{ noCapsulesFound ? katelloHostname : hostname(selectedCapsule.url) }}/pub/{{ consumerCertRPM }}</code></pre>
+        <pre><code>rpm -Uvh https://{{ noCapsulesFound ? katelloHostname : hostname(selectedCapsule.url) }}/pub/{{ consumerCertRPM }}</code></pre>
       </li>
       <li>
-        <p translate>Run subscription-manager in a console on the client host.  You may use an Activation Key to register:</p>
+        <p translate>Register using subscription-manager:</p>
+        <p translate>Using an Activation Key:</p>
         <pre><code>subscription-manager register --org="{{ organization.label }}" --activationkey="&ltActivation Key Name&gt"</code></pre>
-        <p translate>or authenticate with a username and password:</p>
+        <p translate>Using a username and password:</p>
         <pre><code>subscription-manager register --org="{{ organization.label }}" --environment="Library"</code></pre>
       </li>
       <li>
-        <p translate>Install katello-agent for remote actions and displaying errata information:</p>
+        <p translate>Install client package:</p>
+        <p translate>To only report package, errata & tracer information:</p>
+        <pre><code>yum -y install katello-host-tools</code></pre>
+        <p translate>For remote actions and reporting package, errata & tracer information:</p>
         <pre><code>yum -y install katello-agent</code></pre>
+        <p translate>Learn more about these packages in the <a href="{{ katello_documentation_url('/installation/clients.html') }}">Client Documentation</a> </p>
```

Comment:
this  doc link doesn't actually work, im not sure how to get the bastion pages to use ruby helpers?

Reply:
Anything in this file is using javascript. Put your helpers [here](https://github.com/Katello/katello/blob/master/engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content-host-register.controller.js), I believe.